quartz/components/scripts/explorer.inline.ts

```diff
@@ -113,6 +127,13 @@ function setupExplorer() {
 window.addEventListener("resize", setupExplorer)
 document.addEventListener("nav", () => {
   setupExplorer()
+
+  const explorerContent = document.getElementById("explorer-ul")
+  // select last item in explorer (folder or item)
+  const lastItem = explorerContent?.firstChild?.firstChild?.firstChild?.firstChild?.lastChild
```

Comment:
wait wtf lol you should be able to do this with a `document.querySelector()` with `:last-child`

Reply:
hm not getting it to work this way. tried doing 
```ts
const lastItem = document.querySelector("#explorer-ul > div > div > div > ul:last-child")
```
but the `:last-child` selector doesn't seem to work here as it returns the `<ul>` element itself instead of its child (tried without `>` as well, doing `> ul > div` manually at the end works and returns first div child as expected)

dont think this solution is the cleanest either way, but the first element containing folders/items is also dynamically generated and cant be assigned a static id (unless doing something like `id={'node.depth === 0 ? "root" : ""'}`, but that doesnt seem clean either as it assigns a bunch of `""` ids. though this could be solved with assigning a class, im not sure if this approach would even help)

Reply:
I'm hesitant to hardcode the DOM hierarchy like this hmm I wonder if we can insert a pseudo element at the end of the list as a marker and use that instead

Reply:
ahh amazing idea, didnt even think about doing that.

works using a pseudo element now, much cleaner this way (682f19771e3e11c05587bcbad37236cdd5ec4fce)